migrate/20240722_add_project_reputation.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_enum(:project_reputation, %w[new verified])
+
+    alter_table(:project) do
+      add_column :reputation, :project_reputation, default: "new", null: false
+    end
+
+    run "UPDATE project SET reputation = 'verified'"
```

Comment:
We have existing projects without any payment or accounts known for fraud. We can update this value using the `invoice` table.